rfc021-platform-support-policy.md

```diff
@@ -5,109 +5,17 @@ Status: Accepted
-
-Platform | Versions | Architectures | Package Format | Built on
---- | --- | --- | --- | ---
-AIX | 6.1, 7.1, 7.2 | ppc64 | bff | AIX 6.1
-CentOS | 5, 6, 7 | i386, x86_64 | rpm | RHEL 5
-Cisco IOS XR | 6 | x86_64 | rpm | Cisco IOS XR 6 Developer Image
-Cisco NX-OS | 7 | x86_64 | rpm | Cisco NX-OX 7 Developer Image
-Debian | 7, 8 | i386, x86_64 | deb | Debian 6
-FreeBSD | 9, 10 | i386, amd64 | pkg_add pkg | FreeBSD 9
-Mac OS X | 10.9, 10.10, 10.11 | x86_64 | dmg | Mac OS 10.9
-Oracle Enterprise Linux | 5, 6, 7 | i386, x86_64 | rpm | RHEL 5
-Red Hat Enterprise Linux | 5, 6, 7 | i386, x86_64 | rpm | RHEL 5
-Solaris | 10, 11 | sparc, x86 | shar | Solaris 10
-Windows | 7, 8, 8.1, 2008, 2008R2, 2012, 2012R2 | x86, x86_64 | msi | Windows 2008R2
-Ubuntu Linux | 12.04, 14.04 | x86, x86_64 | deb | Ubuntu 12.04
-
-### Tier 2 Support
-
-Tier 2 supported platforms are those on which Omnitruck will serve packages, but those packages may not have been built on that OS variant. Additionally, we may or may not do post-build verification on these platforms. A platform must have at least one Maintainer before being added to this tier.
-
-* SUSE Linux Enterprise Server 11, 12
-* Scientific Linux 5.x, 6.x and 7.x (i386 and x86-64)
-* Fedora (current non-EOL revisions)
-* OpenSUSE 13.1/13.2/42.1
-* OmniOS stable and LTS releases
-
-### Not Supported
-
-"Not supported" means there may be code in-tree, but we don't build for and test on those platforms. At our discretion, we may take patches that don't break any tier 1 or tier 2 platforms, but we have no way of testing these.
-
-* Solaris < 10
-* AIX 5.1L
-* FreeBSD 8
-* NetBSD
-* OpenBSD (i386 and amd64)
-* Windows 2003R2, Windows 2003, Windows 2000
-* RHEL/CentOS/Oracle/Scientific 4.x or older
-* RHEL or SLES on POWER (ppc64) or System/z
-* HP-UX
-* Mac OS X < 10.9, anything ppc
-* Debian < 7.0, anything ppc or arm
-* Gentoo Linux (rolling release)
-* Arch Linux (rolling release)
-
-## Chef Server
-
-Includes any of the add-ons (webui2/manage, push, etc.)
-
-### Supported
-
-* Ubuntu 12.04LTS, 14.04LTS
-* RHEL 5.x, 6.x, 7.x
-* CentOS 5.x, 6.x, 7.x
-* Oracle Enterprise Linux 5.x, 6.x, 7.x
-
-### Unsupported
-
-* Any other Linux or UNIX distributions
-* Windows
-
-## ChefDK
-
-### Supported
-
-* Windows 7, 8, 8.1
-* Fedora (current non-EOL releases)
-* RHEL 6.x, 7.x
-* Mac OS X 10.9, 10.10, 10.11
-* Ubuntu 12.04, 14.04
-
-ChefDK bundles Chef Client. Therefore, Chef Client is supported, by extension, on the foregoing client platforms, if not already mentioned explicitly in the Chef Client support matrix.
-
-### Unsupported
-
-* Windows Vista, XP, 2000
-* Mac OS X < 10.9, anything ppc
+The [chef docs site](https://docs.chef.io/) has [details of current adopted platforms](https://docs.chef.io/platforms.html).
```

Comment:
I'd say something like: 

The Chef Docs site contains the list of platforms that Chef Software currently builds packages for and tests.

The list of working platforms maintained by the Chef community is here:

blah blah blah raspbian blah blah.